src/editors/containers/VideoEditor/components/VideoSettingsModal/components/DurationWidget.jsx

```diff
@@ -49,28 +48,30 @@ export const DurationWidget = ({
     >
       <FormattedMessage {...messages.durationDescription} />
       <Row className="mt-4">
```

Comment:
If you want all the alignments to match, you can change `Row` to `Form.Row` but that is not necessary for approval.

Reply:
Good catch. I'll make that change